src/routes/submit-route/submitted-slops/submitted-list.jsx

```diff
@@ -0,0 +1,104 @@
+import { useQuery } from "@apollo/client";
+import {
+  Button,
+  DeleteMovieModal,
+  MovieCard,
+  MoviePreviewModal,
+} from "../../../components/";
+import { GET_MOVIES } from "../../../graphql";
+import checkIcon from "../../../images/check-mark-dark.svg";
+import pencilIcon from "../../../images/pencil.svg";
+import deleteIcon from "../../../images/red-x-button.svg";
+import { useModals } from "../../../store";
+
+export const SubmittedList = ({ currentUser }) => {
+  const { openModal, closeModal } = useModals();
+
+  // Determine the data query based on the user's admin status
+  const queryVariables =
+    currentUser.isAdmin && currentUser
+      ? {
+          where: {
+            status: {
+              equals: "draft",
+            },
+          },
+        }
+      : {
+          where: {
+            author: {
+              id: {
+                equals: currentUser.id,
+              },
+            },
+            status: {
+              equals: "draft",
+            },
+          },
+        };
+
+  // Use the query with the determined variables
+  const { data, loading, error } = useQuery(GET_MOVIES, {
+    variables: queryVariables,
+  });
+
+  const handleApproveClick = (movieData) => {
+    openModal(
+      <MoviePreviewModal
+        closeModal={closeModal}
+        whiteButton
+        publishButtons
+        selectedMovie={movieData}
+      />
+    );
+  };
+
+  function handleDeleteClick(movieData) {
+    openModal(
+      <DeleteMovieModal
+        movieData={movieData}
+        closeModal={closeModal}
+        currentUser={currentUser}
+      />
+    );
+  }
+
+  return (
+    <div className="w-fit h-fit">
+      {!loading && (
+        <div className="flex flex-col gap-y-5">
+          {data.movies.map((movie) => (
```

Comment:
![image](https://github.com/jahorwitz/slopopedia/assets/110426130/1d71bee6-aeb8-4572-8cc7-4ffb9560e42f)
I would add a question mark here since data is not initialized at first.

Reply:
I would also recommend fixing the merge conflicts